services/app/src/contexts/ChessBoardContext.jsx

```diff
@@ -27,6 +28,11 @@ const ChessBoardContextProvider = ({ children }) => {
     setBoard(copyOfBoard);
   };
 
+  useEffect(() => {
```

Comment:
I think `useMemo` would fit better here:
```
const canPlay = useMemo(() => players.length > 0, [players]);
```
I'm not sure if the `playerDeleted` needs to be included in the dependencies. It is included in the `useEffect` deps, but I feel like if the player is deleted, the `players` list should be updated as well, so the `playerDeleted` might be redundant